packages/components/src/badge/index.tsx

```diff
@@ -3,8 +3,8 @@ import type { ReactNode } from 'react';
 
 import './style.scss';
 
-type BadgeProps = {
-	type:
+export type BadgeProps = {
+	type?:
 		| 'warning'
 		| 'warning-clear'
 		| 'success'
```

Comment:
Rather than using `BadgeProps['type']` in other components, might it be clearer to have a dedicated type for just the value?
```suggestion
export type BadgeType = 'warning' | 'warning-clear' | 'success' | ...;
export type BadgeProps = {
	type?: BadgeType
```